### ISSUE TITLE
has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource

### ISSUE THREAD

**Issue by Ahmad-Fathy** [laravel-filament/filament#5663]
### Package

filament/filament

### Package Version

v2.0.0

### Laravel Version

v9.19

### PHP Version

PHP 8.2

### Problem description

I am using Spatie Media Library to upload files to google drive using the package 

[https://github.com/yaza-putu/laravel-google-drive-storage](url)

every thing works as expected so i can store pdf file at Google Drive and even open it in a new tab, except that i can't retrive the file in the form as shown in the video 

[loading indicator](https://clipchamp.com/watch/vK8QEE91KSD)

Error at the console:

Access to fetch at 'https://drive.google.com/uc?id=1xqsvuarL9pRTvnRIALMJkJTqOew8pTCr&export=media' from origin 'http://citizens.net' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

### Expected behavior

get the file from google drive to the file upload input so i can change or delete it 

### Steps to reproduce

when i use this crome extension

https://webextension.org/listing/access-control.html

and enable CORS Unblock .. it works

### Reproduction repository

https://github.com/spatie/laravel-medialibrary/tree/main

**Comment by danharrin** [comment]
No reproduction repo... please read the instructions when creating a new issue. Also, I think this is something you will need to fix in a PR yourself, we support S3-compatible filesystem disks, it would be such a huge task to support every 3rd party disk in existence.

**Comment by barakhubs** [comment]
solved mine by adding <IfModule mod_headers.c>
    Header set Access-Control-Allow-Origin "http://your-domain-here.com"
    Header set Access-Control-Allow-Credentials "true"
    Header set Access-Control-Allow-Methods "GET, POST, OPTIONS"
    Header set Access-Control-Allow-Headers "DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Authorization"
</IfModule>
 in the htaccess

**Comment by ricventu** [comment]
Hi,
i solved setting in livewire config `temporary_file_upload.disk` to `local`

```sh
php artisan livewire:publish --config
```

```php
    'temporary_file_upload' => [
        'disk' => 'local',
        ...
```
